### PR TITLE
feat: urange syntax

### PR DESCRIPTION
**Pull request by alexander-akait**
**Description:**

Feat parse `<urange>` syntax https://www.w3.org/TR/css-syntax-3/#urange-syntax and no errors on this anymore

**BREAKING CHANGE:**

Yes, new types

**Related issue (if exists):**

No

### CODE REVIEW

File: crates/swc_css_parser/src/parser/value/mod.rs [hunk 1]
```diff
@@ -1326,6 +1330,214 @@ where
     }
 }
 
+// <urange> =
```
Comment (kdy1) [inline crates/swc_css_parser/src/parser/value/mod.rs:1333]: I love this comment

File: crates/swc_css_parser/src/parser/value/mod.rs [hunk 2]
```diff
@@ -1326,6 +1330,214 @@ where
...
+                            _ => {
+                                unreachable!();
+                            }
+                        };
+
+                        urange.push(question);
+                    }
+                } else {
+                    let question = match bump!(self) {
+                        Token::Delim { value } => value,
```
Comment (kdy1) [inline crates/swc_css_parser/src/parser/value/mod.rs:1404]: I think we should check if it's `?`?

**Review by kdy1 — CHANGES_REQUESTED** [review]
Thanks!

File: crates/swc_css_parser/src/parser/value/mod.rs (hunk 1 above)
Comment (alexander-akait) [reply crates/swc_css_parser/src/parser/value/mod.rs:1333]: In future I will add more these comments, it allow to read code without looking on strange logic inside :star:

File: crates/swc_css_parser/src/parser/value/mod.rs (hunk 2 above)
Comment (alexander-akait) [reply crates/swc_css_parser/src/parser/value/mod.rs:1404]: Good catch, fixed

**Comment by alexander-akait** [comment]
@kdy1 Fixed and rebased

**Review by kdy1 — APPROVED** [review]
Perfect! Thanks!

**Pull request merged**